requirements.txt

```diff
@@ -12,10 +12,11 @@ click==8.1.3
 cloudpickle==2.2.0
 colorama==0.4.4
 contourpy==1.0.6
+coverage==6.5.0
```

Comment:
chyba trzeba dodać jeszcze pytest do requirements

Reply:
To jest wbudowana w pythona biblioteka, nie ma potrzeby

Reply:
Chodziło Ci o `pyunit`?